src/Interface/ComputationalRelation.agda

```diff
@@ -75,71 +137,72 @@ module _ {STS : C → S → Sig → S → Set} (comp : Computational STS) where
   Computational⇒Dec : ⦃ _ : DecEq S ⦄ → Dec (STS c s sig s')
   Computational⇒Dec {c} {s} {sig} {s'}
     with compute c s sig | ExtendedRel-compute {c} {s} {sig}
-  ... | nothing | ExSTS = no (ExSTS s')
-  ... | just x  | ExSTS with x ≟ s'
+  ... | failure _ | ExSTS = no (ExSTS s')
+  ... | success x  | ExSTS with x ≟ s'
   ... | no ¬p    = no  λ h → ¬p $ sym $ computational⇒rightUnique h ExSTS
   ... | yes refl = yes ExSTS
 
-module _ {STS : C → S → Sig → S → Set} (comp comp' : Computational STS) where
+module _ {STS : C → S → Sig → S → Set} (comp comp' : Computational STS Err) where
 
   open Computational comp  renaming (compute to compute₁)
   open Computational comp' renaming (compute to compute₂)
 
-  compute-ext≡ : compute₁ c s sig ≡ compute₂ c s sig
-  compute-ext≡ = ExtendedRel-rightUnique comp
-    (ExtendedRel-compute comp) (ExtendedRel-compute comp')
+  --compute-ext≡ : compute₁ c s sig ≡ compute₂ c s sig
```

Comment:
If you replace `≡` by the equality for `ComputationalResult`, this should still hold